### DISCUSSION TITLE
关于VcMeasurements的组件在自己单独写的vue子组件中引入然后在父组件引入该子组件。不显示

### DISCUSSION THREAD

**Discussion by HHANG52** [zouyaoji/vue-cesium#433] [Q&A] [answered]
父组件中引入自己的box
 `//其余标签省略
<vc-viewer>    
  //子组件
<box> </box>
</vc-viewer>`

box 子组件引入VcMeasurements
` //其余标签省略
<vc-measurements style="position: absolute "
                           ref="measurementsRef3"
                           position="top-left"
                           :main-fab-opts="measurementFabOptions3"
                           :distance-measurement-opts="distanceMeasurementOpts3"
                           :component-distance-measurement-opts="componentDistanceMeasurementOpts3"
                           :point-measurement-opts="pointMeasurementOpts3"
                           :editable="true">
       </vc-measurements>`
这样不能让页面显出这个测量工具了。有什么解决办法吗，还是就不支持这样操作。

**Comment by zouyaoji** [comment]
是支持的哈，为啥出不来还得你提供复现的我看看。

**Comment by HHANG52** [comment]
感谢百忙之中回复，辛苦
子组件
![屏幕截图 2023-08-24 171402](https://github.com/zouyaoji/vue-cesium/assets/81608081/de40086f-a6b3-46b3-b942-1ba6bd5230aa)
在父组件调用
![屏幕截图 2023-08-24 171457](https://github.com/zouyaoji/vue-cesium/assets/81608081/8d037fa3-ea89-403e-ade3-17325ca86b4c)
在页面有占位，不出现
![屏幕截图 2023-08-24 171708](https://github.com/zouyaoji/vue-cesium/assets/81608081/ff21df2a-0c3c-4d3e-b063-5e6e5ee64549)

**Comment by HHANG52** [comment, marked as answer]
解决了，在elementUI组件里给一个div包着这个组件就出现了。
